Exdi/exdigdbsrv/ExdiGdbSrv/LiveExdiGdbSrvServer.cpp

```diff
@@ -1861,6 +1861,14 @@ ADDRESS_TYPE CLiveExdiGdbSrvSampleServer::ParseAsynchronousCommandResult(_Out_ D
                     stopReply.currentAddress = m_lastPcAddress;
                     *pProcessorNumberOfLastEvent = pController->GetLastKnownActiveCpu();
                     isWaitingOnStopReply = false;
+                } 
+                // Is it an "OK" response w/o any other field (e.g. OpenOCD can send "OK" after 's'/'g')?
+                else if (stopReply.status.isCoreRunning)

```

Comment:
Is this change intentional? Can we take this out and apply this in its own change?